src/pose_calculations.cpp

```diff
@@ -151,22 +156,22 @@ pose_t transform_pose(const CoordinateSystem co,
         {
             case CoordinateSystem::ENU:
                 // Transform to ENU
-                result.x = z_copy;
-                result.y = x_copy;
+                result.x = +cos(co_north_angle) * z_copy + sin(co_north_angle) * x_copy;
+                result.y = -sin(co_north_angle) * z_copy + cos(co_north_angle) * x_copy;
```

Comment:
Are you certain about the sign of the sin here? In my head a 2d rot matrix should be 

[[cos(theta)     -sin(theta)]
 [sin(theta)       cos(theta)]]

which would result in opposite signs of the sines.

Reply:
I have tested it, this is correct. But the fact that we're doing rotation around y and rotating the y-up coordinates to ENU in one makes it a bit unreadable